tutor/src/models/chapter-section.ts

```diff
@@ -68,7 +70,6 @@ export default class ChapterSection extends BaseModel {
         return reduceRight(
             this.asArray,
             (memo:number, num: number, index: number) => memo + (num * Math.pow(100, index + 1)),
-            0,
```

Comment:
The original file didn't have this and it caused the spec to fail by adding trailing zeroes to the output.

Reply:
nice! ts caught what was probably a bug there